src/test/java/org/jclouds/vcloud/director/v1_5/compute/strategy/VCloudDirectorComputeServiceContextLiveTest.java

```diff
@@ -67,23 +65,26 @@ public void testLaunchNode() throws RunNodesException {
               .locationId("https://emea01.canopy-cloud.com/api/vdc/e931a09d-131f-4aaa-a667-efbe02eba428") // TAI2.0
               //.imageNameMatches("centos6.4x64") // TAI
               .imageNameMatches("CentOS_66_x64_platform") // TAI2.0
+              .minCores(4)
+              .minRam(2048)
               .build();
       // test passing custom options
       VCloudDirectorTemplateOptions options = template.getOptions().as(VCloudDirectorTemplateOptions.class);
 
-      options
+      options.memory(1024);
+      // .networks("Operational_Network_01"); // TAI2.0
       //.networks("Deployment_Network_01"); // TAI
-      //.networks("Operational_Network_01"); // TAI2.0
-              .networks("Operational_Network_01"); // TAI2.0
+
+      options.networks("Operational_Network_01"); // TAI2.0
       NodeMetadata node = null;
       try {
          Set<? extends NodeMetadata> nodes = context.getComputeService().createNodesInGroup(name, 1, template);
          node = Iterables.getOnlyElement(nodes);
-         logger.debug("Created Node: %s", node);
-         SshClient client = context.utils().sshForNode().apply(node);
-         client.connect();
-         ExecResponse hello = client.exec("mount");
-         logger.debug(hello.getOutput().trim());
+//         logger.debug("Created Node: %s", node);
```

Comment:
Worth a comment to explain why commented out (e.g. because VMs just have private IP, so would need a NAT rule to reach ssh port)?